config/database.yml

```diff
@@ -1,20 +1,25 @@
-# SQLite version 3.x
-#   gem install sqlite3
-#
 #   Ensure the SQLite 3 gem is defined in your Gemfile
 #   gem 'sqlite3'
-development:
-  adapter: postgresql
-  encoding: utf-8
-  database: collated_areas_development
+#
+default: &default
+  adapter: postgis
   pool: 5
-  username: postgres
-  password:
+  timeout: 5000
+  host: <%= ENV['POSTGRES_HOST'] %>
+  username: <%= ENV['POSTGRES_USER'] %>
+  password: <%= ENV['POSTGRES_PASSWORD'] %>
+  template: template_postgis
 
+development:
+  <<: *default
+  database: protectedplanet-db
```

Comment:
Can you also keep the name consistend? I would say `pp_development`

Reply:
Sorry @Levia this again is my Docker specific settings which I committed by mistake.